components/uuf-renderablecreator-hbs/src/main/java/org/wso2/carbon/uuf/renderablecreator/hbs/impl/js/JsFunctionsImpl.java

```diff
@@ -159,10 +160,13 @@ public DestroySessionFunction getDestroySessionFunction() {
 
     public SendToClientFunction getSendToClientFunction() {
         if (sendToClientFunction == null) {
-            sendToClientFunction = (name, value) -> {
-                String scriptTag = "<script type=\"text/javascript\">var " + name + "=" + GSON.toJson(value) +
+            sendToClientFunction = (name, values) -> {
+                String scriptTag = "<script type=\"text/javascript\">var " + Handlebars.Utils.escapeExpression(name) +
+                        "=" + GSON.toJson(values[0]) +
                         ";</script>";
-                api.getRequestLookup().addToPlaceholder(Placeholder.js, scriptTag);
+                api.getRequestLookup().addToPlaceholder(
+                        (values.length == 2 && "headJs".equalsIgnoreCase((String) values[1])) ? Placeholder.headJs :
```

Comment:
Is this actually a requested feature? why would somebody want to specify the the place holder name.